content/en/docs/project-user-guide/application-workloads/deployments.md

```diff
@@ -7,4 +7,291 @@ description: 'Kubernetes Deployments'
+
+  - Pod Decentralized Deployment is mean anti-affinity
+  - Pod Aggregation Deployment is mean affinity
+
+  For more information about affinity and anti-affinity, please visit  [Pod affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity)
+
+### Step 4: Input Volumes Data
+
+![deployments](/images/docs/project-user-guide/workloads/deployments_form_3.png)
+
+Mount Volume has 3 methods:
+
+- **Add Volume**: Support EmptyDir and PersistentVolumeClaim.
+
+  In **Add Volume** there are 3 kinds of volumes:
+
+  - **Existing Volume**: Use *PVC* to mount.
+
+    Persistent storage volumes can be used to save user's persist data, Add Existing Volume means you need to create volumes in advance
+
+  - **Temporary Volume**: Use *emptyDir* to mount.
+
+    The temporary storage volume represents [emptyDir](https://kubernetes.cn/docs/concepts/storage/volumes/#emptydir), which is first created when a Pod is assigned to a Node, and exists as long as that Pod is running on that node. When a Pod is removed from a node for any reason, the data in the emptyDir is deleted forever.
+
+  - **HostPath**: Use *HostPath* to mount.
+
+    A HostPath volume mounts a file or directory from the host node's filesystem into your Pod. This is not something that most Pods will need, but it offers a powerful escape hatch for some applications.
+
+- **Mount ConfigMap or Secret**: Supports for configuring the key-value pairs ​​in ConfigMap or Secret via reference configuration center.
+
+  A secret volume is used to pass sensitive information, such as passwords, to Pods. Secret volumes are backed by tmpfs (a RAM-backed filesystem) so they are never written to non-volatile storage.
+
+  ConfigMap is used to store configuration data in the form of key-value pairs. The configMap resource provides a way to inject configuration data into Pods. The data stored in a ConfigMap object can be referenced in a volume of type configMap and then consumed by containerized applications running in a Pod.
+
+  - Set the value of the environment variable
+  - Set command parameters in the container
+  - Create a config file in the volume
+
+For more information about volume, please visit [Volumes](../../storage/volumes)
+
+{{<notice tip>}}
+
+  Deployments can't use the volume template, it is different from StatefulSet.
+
+{{</ notice>}}
+
+### Step 5: Input Advanced Data
+
+![deployments](/images/docs/project-user-guide/workloads/deployments_form_4.png)
+
+- **Set Node Scheduling Policy**
+
+  You can allow Pod replicas to run on specified nodes. It is used `nodeSelector`.
+
+- **Add Metadata**
+
+  Additional metadata settings for resources such as **Labels** and **Annotations**.
+
+## Check Deployment Detail
+
+You can check the deployment detail via click deployment's name on the list.
```

Comment:
->
You can check the Deployment detail by clicking the Deployment's name in the list.